source/run-commands.txt

```diff
@@ -47,6 +47,22 @@ the server.
    provides consistent behavior by always throwing an exception in these
    scenarios.
 
+Shell Commands
```

Comment:
I think "shell commands" is easy to conflate with "shell methods". What do you think about making the heading the specific task this command accomplishes, like "Clear ``mongosh`` Console" (or something like that), and explaining the ``cls`` method that way.

If we end up with more commands that would fit in here, we can consider a table structure at a later time.

Reply:
Done.